### PR TITLE
07 support

### PR DESCRIPTION
**Pull request by SimonDanisch**
works on top of @RalphAS PR and removes the Image dependency for now.
Pretty sure it shouldn't be needed, not sure if it might be needed for e.g. show + png mine and IJulia ;)

### CODE REVIEW

**Comment by rdeits** [comment]
We intentionally added the dependency on `Images.jl` because when `Image.jl` isn't loaded we end up trying to display `Matrix{<:Colorant}` as a giant SVG: https://github.com/JuliaImages/Images.jl/issues/676#issuecomment-343373351 . I think it would be good to keep that dependency unless that display code has moved to ImageCore now.

**Comment by SimonDanisch** [comment]
Yeah I saw that... So my favorite solution would be to move that code to ImageCore!
@timholy, do you see any problem with that?!

**Comment by timholy** [comment]
Sorry for the delay, I've been swamped with JuliaCon preparations. I seem to remember some attempts to do this before, but I can't remember the issue. If a search of issues in Images doesn't turn something up, someone could just try.

**Comment by juliohm** [comment]
Yes, it would be really nice to have Images.jl knocked out from the dependency list in favor of something smaller like ImageCore.jl What is the status on this?

**Comment by juliohm** [comment]
Just sharing that the package QuartzImageIO.jl does it properly on Mac, and only depends on ImageCore.jl: https://github.com/JuliaIO/QuartzImageIO.jl

Could you please share the current plans? If the present PR could be merged and tagged, that would be even better. Updates for VisualRegressionTests.jl depend on this: https://github.com/JuliaPlots/VisualRegressionTests.jl/pull/12

**Comment by Evizero** [comment]
This sounds like an important discussion to be had, but I would like to make the case that at this point we should prioritize getting the Images ecosystem to work.

This PR passes locally for me. can we merge?

**Comment by SimonDanisch** [comment]
I'm rebuilding the ImageMagick binaries to include the 0.7 windows fix

**Comment by Evizero** [comment]
We are getting close 🎉 . ~~After we merge and tag https://github.com/JuliaImages/ImageMetadata.jl/pull/30~~ (**EDIT: done**), this PR is the next element of the chain.

So after some version of this PR is merged and ImageMagick.jl tagged, we can tag the following chain

- TestImages
  - Images
  - ImageInTerminal 
    - ReferenceTests 
      - couple of other packages I maintain

I don't know about ImageView and other JuliaImages packages that are not Images.jl dependencies.

File: REQUIRE [hunk 1]
```diff
@@ -1,7 +1,6 @@
-julia 0.6
-Compat 0.24
+julia 0.7-beta
```
Comment (Evizero) [inline REQUIRE]: Should we just go with `julia 0.7` at this point?  I think thats encouraged after the first release version is out

**Comment by Evizero** [comment]
One behaviour that is different after this PR is that if someone does

```
using FileIO
load("myimg.png")
```

inside a notebook, it fill not restrict and probably be slow (unless the user has `using Images` somewhere).

**Comment by SimonDanisch** [comment]
It should not show it at all, right?

**Comment by SimonDanisch** [comment]
image

**Comment by Evizero** [comment]
oh good, seems like my memory of what would happen is outdated

**Comment by Evizero** [comment]
travis is passing!

looks like the appveyor.yml needs updates (see https://discourse.julialang.org/t/ann-improved-appveyor-scripts/12754 )

**Comment by juliohm** [comment]
Awesome, all tests are passing :100:

**Comment by andreasnoack** [comment]
Any objections to merging this?

**Pull request merged**